.gitignore

```diff
@@ -1,33 +1,34 @@
-*~
```

Comment:
Why do you reorder gitignore file?
If you need to add something -- please do it explicitly.
I just do miss your changes.